plugins/gzip/gzip.cc

```diff
@@ -480,8 +480,8 @@ gzip_transformable(TSHttpTxn txnp, bool server, HostConfiguration *host_configur
 
   resp_status = TSHttpHdrStatusGet(bufp, hdr_loc);
 
-  // conservatively pick some statusses to compress
-  if (!(resp_status == 200 || resp_status == 404 || resp_status == 500)) {
+  // conservatively pick some statusses to compress. ToDo: Make this configurable?
+  if (resp_status != 200) {
```

Comment:
@oschaaf  It is *possible*, but unlikely. The immediate issues we're having is that gzip works very poorly together with the escalate plugin, but you wouldn't escalate on a 200 OK. Long term, @meeramn is going to add a new plugin API, that enforces uniqueness on the continuations added to a hook.